db/migrate/20170519123223_add_request_uuid_to_audits.rb

```diff
@@ -0,0 +1,10 @@
+class AddRequestUuidToAudits < ActiveRecord::Migration
+  def self.up
+    add_column :audits, :request_uuid, :string
```

Comment:
I'm not sure where the audit documentation actually is. There's [this](https://github.com/collectiveidea/audited) which looks very similar but isn't guaranteed to work on even Rails 4.0 and it seems to *think* further changes should be necessary for audits to work… but they don't seem to be…?